types.go

```diff
@@ -48,12 +48,14 @@ const (
 	LineTable
 	// Line numbers table TD wrapper style.
 	LineTableTD
-	// Input that could not be tokenised.
```

Comment:
This is not a spelling mistake...

Reply:
Yeah, I looked it up when I was working on this PR and both spellings are valid - I guess I must have changed it here without checking though.